src/andresmoschini/StringCalculator.test.ts

```diff
@@ -14,12 +14,7 @@ describe("StringCalculator.add", () => {
     expect(result).toEqual(expectedResult);
   });
 
-  it.each([
-    { input: "   " },
-    { input: "abc" },
-    { input: ",5,6,7" },
-    { input: "x5x6x7x" },
-  ])(
+  it.each([{ input: "   " }, { input: "abc" }, { input: "x5x6x7x" }])(
```

Comment:
@andresmoschini I think that the code indentation changes are not in the commit scope (_string-calculator(step 3): return the sum of comma delimited values_)
```
it.each([
    { input: "   " },
    { input: "abc" },
    { input: "x5x6x7x" },
  ])(
```

Off-topic: I do not know why the suggestion button is hidden 

![image](https://user-images.githubusercontent.com/75738115/171698622-1cdaad09-c0fc-4273-a458-ed7ebc4a2064.png)

Reply:
> @andresmoschini I think that the code indentation changes are not in the commit scope (string-calculator(step 3): return the sum of comma delimited values)

We are using prettier in all commits, so these kinds of changes are always included in the same commit in our javascript projects.

> Off-topic: I do not know why the suggestion button is hidden

Because it is already merged.

Reply:
> We are using prettier in all commits, so these kinds of changes are always included in the same commit in our javascript projects.

I understand! 👍 

> Because it is already merged.

Thanks! I think that GitHub is wrong here. What is the problem with the suggestion option in a merged PR?

Reply:
I do not know, but I suppose that a suggestion has no sense when the PR was already merged.

![image](https://user-images.githubusercontent.com/1157864/171700791-d95fd604-ceb6-4979-b347-4fcd30352911.png)

You can write it manually anyway:

```suggestion
test
```

Reply:
> You can write it manually anyway:

_hecha la ley hecha la trampa_